configs/mixmim/metafile.yml

```diff
@@ -36,4 +36,4 @@ Models:
     Weights: https://download.openmmlab.com/mmclassification/v0/mixmim/mixmim-base_3rdparty_in1k_20221206-e40e2c8c.pth
     Converted From:
       Code: https://github.com/Sense-X/MixMIM
-    Config: configs/mixmim/mixmim-base-p16_8xb64-pt300e-in1k.py
+    Config: configs/mixmim/mixmim-base_3rdparty_in1k.py
```

Comment:
Well, the model (checkpoint) can be `3rdparty`, but the config file won't be `3rdparty`. Please rename it to `mixmim-base_8xb64_in1k.py`